docs/internal.rst

```diff
@@ -228,6 +230,9 @@ Internal design
 .. figure:: promise.svg
 	    :scale: 80%
 
+.. TODO: 2 ""s in the figure.
+	Q to @masatake-san: Are they typos of "forced"?
```

Comment:
You are correct. They shold be "forced".
I will fix these in a new pull request.

Reply:
You already fixed in #2917.  Thanks.

I thought I had to ask you which SVG tool you were using.  But I've found you are using Inkscape, by seeing the diff.
I will try it.

Reply:
Yes, I'm using Inkscape.